### DISCUSSION TITLE
Error installing vercel for the first time

### DISCUSSION THREAD

**Discussion by Mich45** [vercel/vercel#8103] [Help] [answered]
Hi everyone!

It's my first time installing vercel and I'm having trouble getting it working.

I ran `npm i -g vercel` and got the following error:

```bash
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@vercel%2fhydrogen - Not found
npm ERR! 404
npm ERR! 404  '@vercel/hydrogen@0.0.2' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\A\AppData\Local\npm-cache\_logs\2022-07-08T00_20_46_887Z-debug.log

```
I have tried using Yarn but I get the same error even. 
Any help will be appreciated. Thanks!

**Comment by amyegan** [comment, marked as answer]
Hi @Mich45. Looks like they have this one solved now (https://github.com/vercel/vercel/issues/8104), but let me know if you're still seeing this error.

**Reply by Mich45** [reply]
Thanks @amyegan! Everything works fine now 🙂
